testing/vips/APKBUILD

```diff
@@ -11,7 +11,7 @@ depends_dev="libjpeg-turbo-dev libexif-dev lcms2-dev"
 makedepends="$depends_dev fftw-dev giflib-dev glib-dev libpng-dev libwebp-dev
 	expat-dev orc-dev tiff-dev"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang $pkgname-tools"
-source="https://github.com/jcupitt/libvips/releases/download/v$pkgver/$pkgname-$pkgver.tar.gz"
+source="https://github.com/libvips/libvips/releases/download/v$pkgver/$pkgname-$pkgver.tar.gz"
```

Comment:
Why did you chane the source url?

Reply:
https://github.com/jcupitt/libvips is a fork of https://github.com/libvips/libvips - so it seems this one to be the official one and @jcupitt seem to maintain both

Reply:
Hello, yes, libvips has been moved to it's own org on github and out of my personal space. It's supposed to be part of opening the project up a bit. It now has five maintainers.

I'm keeping a personal fork so that the github.io doc links still work. These are not forwarded automatically when a repo moves, unfortunately.